dist/stat_data.py

```diff
@@ -1010,6 +1010,7 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_eviction_clean', 'unmodified pages evicted'),
     CacheStat('cache_eviction_deepen', 'page split during eviction deepened the tree'),
     CacheStat('cache_eviction_dirty', 'modified pages evicted'),
+    CacheStat('cache_eviction_dirty_obsolete_time', 'modified pages evicted due to obsolete time'),
```

Comment:
Should we replace time with tw for the stat name and say time window in the description?

Reply:
Changed as suggested.